heron/healthmgr/src/java/org/apache/heron/healthmgr/HealthManager.java

```diff
@@ -329,6 +337,8 @@ protected void configure() {
         bind(String.class)
             .annotatedWith(Names.named(CONF_METRICS_SOURCE_TYPE))
             .toInstance(type);
+        bind(HealthManagerMetrics.class)
+            .toInstance(publishingMetricsRunnable);
```

Comment:
Is naming this instance a "Runnable" necessary? What do you think about naming it just `metricsPublisher`.

Reply:
done